wagtail_localize/machine_translators/dummy.py

```diff
@@ -0,0 +1,53 @@
+from bs4 import BeautifulSoup, NavigableString, Tag
+from django.utils.translation import gettext_lazy as _
+
+from wagtail_localize.strings import lstrip_keep, rstrip_keep, StringValue
+
+from .base import BaseMachineTranslator
+
+
+def language_code(code):
```

Comment:
We have two language code functions in this PR. Any chance we could merge them together?

Reply:
All the backends have their own special cases when it comes to handling language codes so I think we should keep them separate